webhooks/networking/ingress_validator.go

```diff
@@ -19,6 +19,9 @@ import (
 
 const (
 	apiPathValidateNetworkingIngress = "/validate-networking-v1-ingress"
+	lbAttrsDeletionProtectionEnabled = "deletion_protection.enabled"
+	schemDefault                     = "internal"
+	defaultIngressClass              = "alb"
```

Comment:
Per the documentation, the default ingress class is the one with the `ingressclass.kubernetes.io/is-default-class` annotation set to `true`. It's not necessarily the one named `alb`.

 I can't find the code that uses that annotation, though.